PendingReleaseNotes

```diff
@@ -5,6 +5,42 @@
   python3.6 as the native python3. see the `announcement <https://lists.fedoraproject.org/archives/list/epel-announce@lists.fedoraproject.org/message/EGUMKAIMPK2UD5VSHXM53BH2MBDGDWMO/>_`
   for more details on the background of this change.
 
+* Because the content of telemetry reports has changed, you will need to
+  either re-opt-in with::
+
+    ceph telemetry on
+
+  You can view exactly what information will be reported first with::
+
+    ceph telemetry show
+
+* The telemetry module now reports more information about CephFS file systems,
```

Comment:
Now that the first bullet point has been dropped, let's adapt this one to be like https://github.com/ceph/ceph/pull/32160

```
* The telemetry module now reports more information.

  We now report more information about CephFS file systems,
  including:
```

AFAICS, the rest is fine.

Reply:
as for this comment, given the current phrasing, I'm not entirely sure how I'd go about it. Given I've dropped the `more information`, and instead simply describe the information provided, figuring out how to phrase it differently is giving me a headache.